lib/transfer_util.cu

```diff
@@ -704,23 +702,24 @@ namespace quda {
 
   template<typename Float>
   void BlockOrthogonalize(ColorSpinorField &V, int Nvec, 
```

Comment:
Ok, I see my trait suggestion breaks down because of the presence of the Laplace operator.  

One further comment here is to include the Laplace option here in the `GPU_STAGGERED_DIRAC` macro, since the Laplace operator is only built if `GPU_STAGGERED_DIRAC` is defined (since that's the macro that enables nSpin=1 blas routines).  To avoid horrible compile time when doing staggered-only multigrid, perhaps it makes sense to also guard the nSpin=4 and nSpin=2 cases  with `GPU_WILSON_DIRAC` which is enabled for all Wilson-like operators.